### DISCUSSION TITLE
Add an "other option" to every quest to indicate incorrect tagging

### DISCUSSION THREAD

**Discussion by smichel17** [streetcomplete/StreetComplete#3336] [Ideas]
In https://github.com/streetcomplete/StreetComplete/issues/3318#issuecomment-927113228, @westnordost wrote,

> Hmm no, we can not add an other-answer option to fix every incorrect tagging, this is what "leave a note" is for.

…what if we did *exactly* that? Add a **single** entry, present for (almost?) **every** quest, "This %s is mapped incorrectly".

- It would create a note, like "can't say".
- SC would avoid creating quests of *any* type at the location of this type of note.
- I imagine this would be useful information for people looking to assess map quality or doing armchair mapping.

**Comment by westnordost** [comment]
> SC would avoid creating quests of any type at the location of this type of note.

What do you mean? This is already the case

**Reply by smichel17** [reply]
Ah, I didn't realize. I thought it only prevented that particular quest from being shown to other mappers. In other words, if I answered "Can't say" for the sidewalk quest and left a note, that it would still ask the surface type quest, etc. But I just checked and it does indeed prevent all quests from being shown there.
